### DISCUSSION TITLE
Enforced PackageRules

### DISCUSSION THREAD

**Discussion by secustor** [renovatebot/renovate#22258] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Sometimes if managing setups in orgs you want to enforce specific settings this is currently not possible people can overwrite settings in local repositories. 

I propose a second set of packageRules which get applied after normal packageRules. 
These "enforcedPackageRules" should be currently only be configurable on an instance level. If https://github.com/renovatebot/renovate/issues/22212 has been implemented this can be extended to this level too. 

If enforced rules are applied, this should be communicated using the DependencyDashboard

```json
{
  "packageRules": [{
    "matchManagers": ["github-actions"],
    "enabled": false
  }],
  "enforcedPackageRules": [{
     "matchManagers": ["github-actions"],
     "enabled": true
}]
```
Will result in an activated `github-actions` manager.

**Comment by viceice** [comment]
:thinking: Can't you set these package rules inside the `force` object on global config?

```json
{
  "force": {
    "packageRules": [{
      "matchManagers": ["github-actions"],
       "enabled": true
     }]
  }
}
```

**Reply by secustor** [reply]
Have not been aware of `force` TBH, but would it overwrite all packageRules or simply appending?

**Reply by rarkins** [reply]
I'd prefer this if it works:

```json
{
    "packageRules": [{
      "matchManagers": ["github-actions"],
      "force": {
         "enabled": true
      }
     }]

}
```

Also assumes we adequately block `force` in repo config!
